src/Resources/config/state_machine/checkout.yaml

```diff
@@ -1,5 +1,9 @@
 winzou_state_machine:
   sylius_order_checkout:
+    transitions:
+      complete:
+        from: [payment_selected, payment_skipped, addressed, shipping_skipped, payment_selected, cart]
+        to: completed
```

Comment:
Overwriting it from the plugin is not a good idea, write it in `tests/Application` and add the instruction to the readme.